### DISCUSSION TITLE
Why the Response fields are not pub but pub(crate)?

### DISCUSSION THREAD

**Discussion by asimaranov** [emilk/egui#555] [Q&A]
Standard widgets e.g. slider use pub(crate) fields of response e.g. ```changed ```
![image](https://user-images.githubusercontent.com/25568730/125188121-4f850e80-e23b-11eb-91e9-cc33be42605e.png)

If I try to do something like that in my code, it obviously fails because fields such as ```changed``` are pub(crate)
![image](https://user-images.githubusercontent.com/25568730/125188189-8fe48c80-e23b-11eb-89f2-e1ee371b16e7.png)

**Comment by parasyte** [comment]
For what reason do you need to mutate the `Response` in this way?
